tests/Aspire.Hosting.Azure.Tests/AzureCosmosDBEmulatorFunctionalTests.cs

```diff
@@ -128,15 +128,15 @@ await pipeline.ExecuteAsync(async token =>
         }, cts.Token);
     }
 
-    [Theory(Skip = "Using CosmosDB emulator in integration tests leads to flaky tests - https://github.com/dotnet/aspire/issues/5820")]
+    [Theory(Skip = "CosmosDB Emulator takes too long to spin up, spinning up twice consecutively will almost always exceed the global test timeout")]
```

Comment:
Use the `[OuterloopTest(..)]` instead of skipping it. That will be then run by `.github/workflows/specialized-test-runner.yml` .